kratos/python_scripts/analysis_stage.py

```diff
@@ -149,6 +150,19 @@ def FinalizeSolutionStep(self):
         for process in self._GetListOfProcesses():
             process.ExecuteFinalizeSolutionStep()
 
+        # self.main_model_part = self.model.GetModelPart(self.project_parameters["solver_settings"]["model_part_name"].GetString())
+        # for element in self._GetSolver().GetComputingModelPart().Elements:
+        #     damage = element.CalculateOnIntegrationPoints(CLA.DELAMINATION_DAMAGE_VECTOR_MODE_ONE,self.main_model_part.ProcessInfo)
+        #     NUM_OF_FAILED_INTEGRATION_POINTS = 0

```

Comment:
DANGEROUS! remove any change in the `analysis stage`

Reply:
Done!